src/web.ts

```diff
@@ -0,0 +1,19 @@
+import { WebPlugin } from '@capacitor/core';
+
+import type { ScreenSizePlugin } from './definitions';
+
+export class ScreenSizeWeb extends WebPlugin implements ScreenSizePlugin {
+
+  async echo(options: { value: string }): Promise<{ value: string }> {
+    console.log('ECHO', options);
+    return options;
+  }
+
+  async getDevicePPI(): Promise<{ density: string }> {
+    console.log('(web) getDevicePPI: not implemented for web');
```

Comment:
maybe better "accessing the PPI is not supported within a browser (web)"?

Reply:
The log is now adjusted according to the example.